security-oauth2/src/test/groovy/io/micronaut/security/oauth2/client/clientcredentials/ClientCredentialsConcurrentSpec.groovy

```diff
@@ -97,9 +98,11 @@ class ClientCredentialsConcurrentSpec extends Specification {
     @AutoCleanup
     BlockingHttpClient client = httpClient.toBlocking()
 
+    @IgnoreIf({ (Runtime.runtime.availableProcessors().intdiv(2) ?: 1) == 1 })
     void "no exception for concurrent requests using client credentials"() {
         when:
-        int numberOfFutures = Runtime.getRuntime().availableProcessors() - 1
+        int numberOfFutures = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
```

Comment:
I am borrowing a page from [Gradle Execute tests in parallel documentation](https://docs.gradle.org/current/userguide/performance.html#execute_tests_in_parallel). This makes this test pass when running in parallel. I have a 10 machine and `Runtime.runtime.availableProcessors()` returns 20.